EDITORS.md

```diff
@@ -88,6 +90,24 @@ lspconfig.ruby_lsp.setup({
 })
 ```
 
+To enable or disable features, you can add an `enabledFeatures` table to the `init_options` options, like so:
```

Comment:
Do we need this section considering we have the list of initialization options?

Reply:
It makes more sense being in the actual Neovim lsp-config docs anyway